functional-tests/scripts/build-dart-functional

```diff
@@ -87,17 +86,8 @@ safe cmake \
     .. > "$tmppipe"
 popd
 
-safe cmake \
-    --build "${BUILD_DIR}" \
-    --target install
-
-LIBRARY_PATH="${INSTALL_DIR}/lib"
-TEST_EXECUTABLE="./bin/test_dart"
+safe cmake --build "${BUILD_DIR}"
 
-pushd "${INSTALL_DIR}"
-if [ "$USE_ASAN" = true ]; then
-    LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.4" LD_LIBRARY_PATH=$LIBRARY_PATH exec $TEST_EXECUTABLE
-else
-    LD_LIBRARY_PATH=$LIBRARY_PATH exec $TEST_EXECUTABLE
-fi
+pushd "${BUILD_DIR}"
+ctest -C Debug --verbose
```

Comment:
Are you sure we're not masking any issues by running a debug build here?

Reply:
Changed script to always use Ninja and ctest doesn't require build config option for it